internal/generate/packagemanifest/packagemanifest.go

```diff
@@ -17,37 +17,60 @@ package packagemanifest
 import (
 	"errors"
 	"fmt"
-	"io"
+	"io/ioutil"
 	"path/filepath"
 	"sort"
+	"strings"
+
+	yaml "gopkg.in/yaml.v1"
 
 	apimanifests "github.com/operator-framework/api/pkg/manifests"
 	"github.com/operator-framework/api/pkg/validation"
 	log "github.com/sirupsen/logrus"
 
 	genutil "github.com/operator-framework/operator-sdk/internal/generate/internal"
-	"github.com/operator-framework/operator-sdk/internal/generate/packagemanifest/bases"
 )
 
 const (
 	// File extension for all PackageManifests written by Generator.
 	packageManifestFileExt = ".package.yaml"
 )
 
+// PackageManifestGenerator is an implementation of the PackageManifestGeneratorSDK interface
+type PackageManifestGenerator struct{}
+
+// GeneratorSDK is an interface that specifies the Generate methods
+// to generate and write various OperatorSDK types such as
+// package manifests
+//go:generate go run github.com/maxbrunsfeld/counterfeiter/v6 . PackageManifestGeneratorSDK
+type PackageManifestGeneratorSDK interface {
+	Generate(operatorName, version, outputDir string, opts ...*PkgOptions) error
+}
+
+// PackageManifest configures the PackageManifest that GetBase() returns.
+type PackageManifest struct {
```

Comment:
This no longer needs to be exported since it is only used within one package. In fact, you can make this into a function `func getBase(packageName, basePath string) (*apimanifests.PackageManifest, error)`